### ISSUE TITLE
Overly large leading in VS 2019 (not VS Code) for 2102.03

### ISSUE THREAD

**Issue by kovesp** [microsoft/cascadia-code#413]
# Environment

```none
Cascadia Code version number: 2102.03
Application (with version) used to display text: Visual Studio 2019 16.8.5
OS platform and version: Windows 10 64 1809
Screen resolution (i.e. 220dpi): Can't find this in Windows. 28" display, 3840x2160 @ 125% scaling, so maybe around 200?

Any other software?
```

# Steps to reproduce
Any editor pane in VS 2019.

# Expected behavior
Single spaced leading, e.g., 10/12  pt.

# Actual behavior
Very large leading, I estimate it to be double spaced lines, or 10/20 pt. Reduces the number of editor lines that are displayed by about half.

# Notes
* Installed Cascadia Code TTF, did not install PL.
* Clean install of VS 2019 on new computer.
* I uninstalled 2102.3 and installed 2009.22. The issue went away.
* The leading for 2103.3 was OK in Visual Studio Code.

**Comment by aaronbell** [comment]
Interesting. There has been no modification of font metrics between 2009.022 and 2102.003. So AFAIK there should be no change in vertical metrics. I'll see if I can repro it.

**Comment by kovesp** [comment]
I did a bit more experimentation and noticed that the problem also shows up in the W10 Font Settings app. Compare the two images for when 2009.22 and 2102.03 are installed.
![image](https://user-images.githubusercontent.com/8108503/107881722-9dc3fa80-6eb3-11eb-8f76-b5070166233e.png)
![image](https://user-images.githubusercontent.com/8108503/107881739-a4eb0880-6eb3-11eb-8a98-f8036b9e6e61.png)

**Comment by arnauldvm** [comment]
I have a similar issue with VSCode terminal.

Had to downgrade to "2009.22" to fix it.

**Comment by aaronbell** [comment]
Hi! Unfortunately I have not been able to repro the issue, either in the W10 font settings, or in VS19.

2009_022:
<img width="329" alt="2009_022" src="https://user-images.githubusercontent.com/8460297/108115536-1e6b2e00-704f-11eb-8400-76dc555fc14d.PNG">

2102_003:
<img width="385" alt="2102_003" src="https://user-images.githubusercontent.com/8460297/108115546-232fe200-704f-11eb-9830-fb51358c5563.PNG">

As I was doing this, something that struck me that may be part of the issue you're running into. With the 2102 release of Cascadia Code, the postscript name of the font changed ("CascadiaCode-Regular" to "CascadiaCode-Roman"). The postscript name is one of the unique names in the font metadata that the computer uses to identify the font file. [Problems can occur](https://github.com/microsoft/cascadia-code/wiki/Double-installation-issue) if there are two fonts installed with the same postscript name, or two fonts installed with the same font name but different postscript names.

So I wanted to ask a couple of things:
(1) Do you have Windows Terminal also installed? That application 'installs' a copy of Cascadia Code / Mono that might be using the old postscript names (if they haven't updated yet, or you haven't updated to the latest). Now, it should only be impacting in Windows Terminal, so this may not be the actual issue. 

(2) Did you try clearing the font cache? If the old version of Cascadia Code was cached, it could be interfering with the new version, causing the bug you're encountering. I'd suggest clearing the cache and restarting the computer. That way, when you downgraded back to the 2009 version, the font once again aligned with the cache version and it appeared to work again. 

Please let me know if you have success!

**Comment by kovesp** [comment]
(1) I don't have Windows Terminal on this machine (much as I would like to :-)), it is running W10 1809.
(2) Tried clearing the font cache, making sure that I rebooted before restating the two caching services. It did not fix the problem. Back to 2009.22 again.

**Comment by aaronbell** [comment]
Dang! Sorry that didn't work out for you. @DHowett, have you or your teammates run into any issues like this?

**Comment by DHowett** [comment]
We haven't seen this one, unfortunately!

**Comment by jnsn** [comment]
I am seeing the same issue with 2102.3, when using the version installed by Chocolatey, also no Windows Terminal has been installed.

**Comment by aaronbell** [comment]
While researching a separate issue, I noted that the `MVAR` table is not properly supported in OS rendering engines and can, under certain circumstances, [lead to abnormal vertical metrics](https://github.com/google/fonts/issues/2085). While I'm not totally sure, I have a suspicion that this might be the cause of the issue that you're running into. 

I've submitted a fix to remove the `MVAR` table as part of PR #410—when it is merged in and the next version of the font is released, please take a look and see if it resolves the spacing for you!

**Comment by kovesp** [comment]
Release 2102.025 fixes the leading problem. Had to clear the Windows font cache. Before that the leading was OK, but ligatures did not work.
Thank you.

**Comment by aaronbell** [comment]
Perfect! I'm glad it solved it for you. Sounds like the `mvar` was indeed the problem.

**Comment by arnauldvm** [comment]
Fixed in 2102.025 for me too.
